.github/workflows/reviewdog.yml

```diff
@@ -19,4 +30,4 @@ jobs:
       - name: shellcheck
         uses: reviewdog/action-shellcheck@v1
         with:
-          github_token: ${{ secrets.github_token }}
+          github_token: ${{ secrets.github_token }}
```

Comment:
Please add a new line to the end of this file.

Reply:
It's done, thaks for pointing out.